runtime/compiler/env/j9method.cpp

```diff
@@ -2141,6 +2141,7 @@ void TR_ResolvedJ9Method::construct()
       {x(TR::java_util_HashMap_findNullKeyEntry,      "findNullKeyEntry",           "()Ljava/util/HashMap$Entry;")},
       {x(TR::java_util_HashMap_get,                   "get",           "(Ljava/lang/Object;)Ljava/lang/Object;")},
       {x(TR::java_util_HashMap_getNode,               "getNode",       "(ILjava/lang/Object;)Ljava/util/HashMap$Node;")},
+      {x(TR::java_util_HashMap_getNode,               "getNode",       "(Ljava/lang/Object;)Ljava/util/HashMap$Node;")},
       {x(TR::java_util_HashMap_putImpl,               "putImpl",       "(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;")},
```

Comment:
as well, same enum name for essentially different recognized methods, any implications? hoped these different methods don't exist at the same time in a single level of language.

Reply:
That was the same question I had, maybe I just asked it in an unclear manner .

Reply:
For this one, I reused it because it was almost the same method and all the locations that used the recognized method wanted to do the exact same thing. But, they aren't completely identical so it might be better to separate them even if the two versions don't ever exist in the same JDK level at the moment.

Reply:
Yes that would be cleaner. Thanks.

Reply:
I have now split off the new `Hashmap.get` to map to `java_util_HashMap_getNode_Object`.

Reply:
Thanks for making that change.  I think people have often reused enumerated values for recognized methods because it was convenient thing to do.  However, it's a potential source of future bugs if someone makes a change to the treatment of a recognized method without realizing that more than one method might be affected.